src/higley_lab_to_nwb/benisty_2024/metadata/benisty_2024_dual_ophys_metadata.yaml

```diff
@@ -0,0 +1,67 @@
+Ophys:
+  Device:
+    - name: TwoPhotonMicroscope
+      description: MOM microscope (Sutter Instruments) coupled to a 16x, 0.8 NA objective (Nikon). Excitation was driven by a Titanium-Sapphire Laser (Mai-Tai eHP DeepSee, Spectra-Physics).
+      manufacturer: Sutter Instruments
+    - name: OnePhotonMicroscope
+      description: Zeiss Axiozoom with a 1x, 0.25 NA objective with a 56 mm working distance. Epifluorescent excitation was provided by an LED bank (Spectra X Light Engine, Lumencor). To image through the implanted prism, a long-working distance objective (20x, 0.4 NA, Mitutoyo) was used.
+      manufacturer: Zeiss
+  TwoPhotonSeries:
+    - name: TwoPhotonSeries
+      description: Imaging data from Green channel recorded with 2p microscope. Images were acquired at 576x576 resolution after 45x pixel binning with 20ms exposure.
+      imaging_plane: TwoPhotonImagingPlane
+      unit: n.a.
+  OnePhotonSeries:
+    - name: OnePhotonSeriesGreenChannel
+      description: Imaging data from Green channel recorded with 1p microscope. Images were acquired at 576x576 resolution after 45x pixel binning with 20ms exposure.
+      imaging_plane: OnePhotonImagingPlaneGreenChannel
+      unit: n.a.
+    - name: OnePhotonSeriesIsosbestic
+      description: Imaging data recorded with 1p microscope for isosbestic control. Images were acquired at 576x576 resolution after 45x pixel binning with 20ms exposure.
+      imaging_plane: OnePhotonImagingPlaneIsosbestic
+      unit: n.a.
+    - name: OnePhotonSeriesRedChannel
+      description: Imaging data from Red channel recorded with 1p microscope. Images were acquired at at 576x576 resolution after 45x pixel binning with 20ms exposure.
+      imaging_plane: OnePhotonImagingPlaneRedChannel
+      unit: n.a.
+  ImagingPlane:
+    - name: TwoPhotonImagingPlane
+      description: Imaging plane for the Green channel recorded with 2p microscope.
+      excitation_lambda: 920.0  # in nm
+      location: Whole Brain
+      device: TwoPhotonMicroscope
+      optical_channel:
+        - name: Green
+          description: Emitted light was collected through a 525/50 filter and a gallium arsenide phosphide photomultiplier tube (Hamamatsu).
+          emission_lambda: 525.0
+      indicator: GCaMP6s #TODO change
+    - name: OnePhotonImagingPlaneRedChannel
+      description: Imaging plane for the Green channel recorded with 2p microscope.
```

Comment:
is it not 1p microscope then?

Reply:
Good Catch! NO it should be 1p, I corrected it